docs/src/domains/cdn/fastly.md

```diff
@@ -1,58 +1,55 @@
-There are only a few notable differences.
-Individual apps may have their own Fastly setup instructions or additional modules.
-Consult the documentation for your app for specific details.
+## Get a DNS TXT record
+
+To use a Fastly CDN included in a Dedicated plan, 
+you need to [request a DNS TXT record from support](../../overview/get-support.md#create-a-support-ticket) before going live.
+To add this DNS TXT record to your domain settings,
+contact your domain registrar.
 
 {{% disable-cache CDN="Fastly" %}}
 
-## Set the Platform.sh domain on Fastly
+## Connect Fastly to an origin server
+
+To allow Fastly to start caching your content,
+you need to connect it to an origin server.  
+
+To do so, follow these steps:
 
-Rather than create a DNS `CNAME` for your default Platform.sh branch (for instance `main-abcd123.abcdefgh1234567.eu.platformsh.site`),
-[configure Fastly](https://docs.fastly.com/guides/basic-configuration/working-with-domains)
-to respond to requests for your domain name and to treat the default Platform.sh branch as its backend server.
-Be sure to enable TLS for the backend connection to Platform.sh.
-Then configure your DNS to point your domain at Fastly instead.
-See the [Fastly documentation](https://docs.fastly.com/guides/basic-configuration/connecting-to-origins) for further details.
+1. [Enable TLS](./_index.md#set-up-tls-certificates) for the backend connection to Platform.sh.
+2. Configure your DNS to point your domain at Fastly.
+   For more information, see your DNS provider's official documentation.
+3. Add your default Platform.sh branch as [a host on Fastly](https://docs.fastly.com/en/guides/working-with-hosts).
 
-## HTTP Redirect
+## Force TLS and enable HSTS
 
-From the Fastly interface under "Configure",
-edit the configuration for your origin to enable the ["Force TLS and enable HSTS"](https://docs.fastly.com/en/guides/enabling-hsts-through-fastly).
+For enhanced security, you can enforce HTTPS communication between your visitors and Fastly,
+and between Fastly and your Platform.sh origin server.   
 
-Generally, Platform.sh recommends specifying only HTTPS routes in your `routes.yaml` file.
-This results in all pages being served over SSL and any requests for an HTTP URL automatically redirected to HTTPS.
-When Fastly has been added as an intermediary to your project, however, this automatic redirect fails.
-Once a user visits your site with an HTTP request and is directed to Fastly,
-Fastly continues to query Platform.sh using HTTPS (since that is how origin has been configured),
-resulting in Platform.sh never detecting the HTTP request in the first place.
-Enabling the "Force TLS and enable HSTS" option causes Fastly to do the same HTTP to HTTPS redirect as Platform.sh,
-ensuring that all requests are over HTTPS end-to-end.
+To do so, from your Fastly account, [force TLS and enable HSTS](https://docs.fastly.com/en/guides/enabling-hsts-through-fastly#forcing-tls-and-enabling-hsts).
```

Comment:
:clipboard: 
```suggestion
To do so, in your Fastly account [force TLS and enable HSTS](https://docs.fastly.com/en/guides/enabling-hsts-through-fastly#forcing-tls-and-enabling-hsts).
```

Reply:
Might also be worth explaining what HSTS is? Not sure, if it's explained enough at the link.